docs.rb

```diff
@@ -12,24 +12,17 @@ def self.plugins
 
 # gets readme files in the folders
 def self.get_readmes
+	c = File.open("docs.md", "w")
 	get_folders = plugins
-	# readmes = []
 	get_folders.each do |folder|
 		Dir.glob('./' + folder + '/*.md') do |md|
 			puts md
 			puts %(md\n=============)
-			File.open("docs.md", "w") do |filea| 
-			File.open(md, "r") do |fileb|
-				fileb.readlines.each do |line|
-				filea.puts line
-			# while line = fileb.gets	
-			# 	filea.puts line 
-			# end 
-			end
-		end
-	end 
-	end 
+			f = File.open(md, "r") 
+				f.each_line do |line|
+					c.write line
```

Comment:
IIUC, this is writing the whole file (or files?) into docs.md?

(sorry, not Ruby savvy)